### ISSUE TITLE
7.9.1 - 500 error in groups/XXX/milestones page

### ISSUE THREAD

**Issue by zisten** [gitlabhq/gitlabhq#9079]
Use GtiLab 7.9.1. When i point to a group's milestornes page , it causes a 500 error.

**Comment by zisten** [comment]
Sorry, 
log here:
```
Started GET "/groups/bioinformatics/milestones" for 127.0.0.1 at 2015-04-02 10:02:21 +0000
Processing by Groups::MilestonesController#index as HTML
Parameters: {"group_id"=>"bioinformatics"}
Completed 500 Internal Server Error in 423ms
ActionView::Template::Error (No route matches {:action=>"show", :controller=>"groups/milestones", :format=>nil, :group_id=>#<Group id: 3, name: "Bioinformatics", path: "bioinformatics", owner_id: nil, created_at: "2014-09-28 10:52:01", updated_at: "2014-12-02 10:36:54", type: "Group", description: "", avatar: "bioinfo.png">, :id=>"", :milestone=>{:state_event=>:close}, :title=>"XXXXX"} missing required keys: [:id]):
24: - if milestone.closed?
25: = link_to 'Reopen Milestone', group_milestone_path(@group, milestone.safe_title, title: milestone.title, milestone: {state_event: :activate }), method: :put, class: "btn btn-sm btn-grouped btn-reopen"
26: - else
27: = link_to 'Close Milestone', group_milestone_path(@group, milestone.safe_title, title: milestone.title, milestone: {state_event: :close }), method: :put, class: "btn btn-sm btn-close"
28: %h4
29: = link_to_gfm truncate(milestone.title, length: 100), group_milestone_path(@group, milestone.safe_title, title: milestone.title)
30: %div
app/views/groups/milestones/index.html.haml:27:in `block in _app_views_groups_milestones_index_html_haml__3717001650072695534_106472140'
app/views/groups/milestones/index.html.haml:20:in `each'
app/views/groups/milestones/index.html.haml:20:in `_app_views_groups_milestones_index_html_haml__3717001650072695534_106472140'
```

**Comment by zisten** [comment]
Seems fixed when upgraded to 7.9.4
